### PR TITLE
fix: a problem when logging the secret value in terraform-provider-csbmssqldbrunfailover

### PR DESCRIPTION
**Pull request by zucchinidev**
Fix a problem when logging the secret value.
We remove the regex introducing less restrictive validations. The validation is on the supplier’s side and we just check if the field is empty or too long.

[#183032358](https://www.pivotaltracker.com/story/show/183032358)
[#183032253](https://www.pivotaltracker.com/story/show/183032253)

### Checklist:

* [x] Have you added Draft Release Notes in `docs/draft-release-notes.md`?
* [x] Have you followed the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)?

### CODE REVIEW

File: docs/draft-release-notes.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## Release notes for next release:
 
 ### New feature:
-
+- Client Secret field validation in failover service: We introduce a less restrictive validation. The main validation is on the supplier's side, and we just check if the field is empty or too long.
```
Comment (blgm) [inline docs/draft-release-notes.md:4]: Are there any edits to these release notes that might be appropriate now that we have new guidance on writing release notes?

File: providers/terraform-provider-csbmssqldbrunfailover/csbmssqldbrunfailover/config.go [hunk 2]
```diff
@@ -7,9 +7,12 @@ import (
...
 )
 
+const (
+	maxLengthClientSecret = 1024
+)
+
 var (
-	identifierRegexp   = regexp.MustCompile(`^[\w_.-]{1,64}$`)
-	clientSecretRegexp = regexp.MustCompile(`^[\w_.-]{1,96}$`)
+	identifierRegexp = regexp.MustCompile(`^[\w_.-]{1,64}$`)
```
Comment (blgm) [inline providers/terraform-provider-csbmssqldbrunfailover/csbmssqldbrunfailover/config.go:15]: We were wondering whether this regular expression might cause problems in the future if there's a valid identifier that doesn't meet our expectations. Should we relax this check too?

**Review by blgm — CHANGES_REQUESTED** [review]
Looks good. It would be worth looking again at the:
- release notes - as there's new guidance on how to write them
- the other Regexp, as we don't want to get another very similar bug

File: docs/draft-release-notes.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ## Release notes for next release:
 
 ### New feature:
-
+- **Failover service parameters validation is more flexible:** Previously, the validation of the parameters was carried out in a more restrictive way adding rigidity to the admitted values. With this change, the main validation is on the supplier side, and we only check if the field is empty.
```
Comment (FelisiaM) [inline docs/draft-release-notes.md:4]: Wouldnt this be a fix rather than new feature?

**Review by blgm — APPROVED** [review]
Looks good. I think I agree with @FelisiaM's comment that the change is a fix rather than a new feature.

File: docs/draft-release-notes.md (hunk 3 above)
Comment (zucchinidev) [reply docs/draft-release-notes.md]: It makes totally sense. I just changed it. Thank you very much @FelisiaM

**Comment by zucchinidev** [comment]
Thanks @blgm  and @FelisiaM . Release notes modified

**Pull request merged**